tests/flamegraph.rs

```diff
@@ -592,3 +592,40 @@ fn load_palette_map_file(palette_file: &str) -> PaletteMap {
     let path = Path::new(palette_file);
     PaletteMap::load_from_file_or_empty(&path).unwrap()
 }
+
+#[test]
+fn flamegraph_unsorted_input_files() {
+    let input_file =
+        "./tests/data/flamegraph/multiple-inputs/perf-vertx-stacks-01-collapsed-all-unsorted-1.txt";
+    let expected_result_file = "./tests/data/flamegraph/unsorted-input/unsorted-input.svg";
```

Comment:
I don't think you need a _separate_ SVG for this. You should be able to just compare it to the one produced from the sorted version of the input, no?

Reply:
The input file is half of a two-part input that's used in the multiple-inputs test. I reused it because it was already unsorted, but it only has half of the stacks of the original test file from upstream. We don't have another SVG to compare to.

The alternative is to create a new input file with all of the stack lines and randomize the order of those, then compare to the one produced by the sorted version like you're suggesting. I guess this option has the added advantage of making sure the outputs of the sorted and unsorted versions always match. I'll change the test to do this.